src/GraphQL.Primitives/GraphQLRequest.cs

```diff
@@ -86,7 +94,7 @@ public override int GetHashCode()
         {
             unchecked
             {
-                var hashCode = Query.GetHashCode();
+                var hashCode = Query?.GetHashCode() ?? 0;
```

Comment:
public override int GetHashCode() => (Query, OperationName, Variables).GetHashCode();

Reply:
That's a neat trick! Are you okay with ignoring all the other keys in the dictionary though? Obviously the original implementation did.

Reply:
I am OK with any options. I just don't like these hand-written methods for a long time.